website/contribute/rfc-process.md

```diff
@@ -0,0 +1,56 @@
+Any of the following can be considered a major change:
+
+ - Any new component, module or code that introduces a new concept into the project or alters the behavior of an existing one
+ - Any large code refactor to address general code re-usability and structure. There is no strong definition for “Large” and whether or not the refactor requires an RFC can be discussed on the @dev mailing list.
+ - Any change that impacts the underlying storage layout/format. e.g changes to the HoodieLogFormat, timeline layout version.
+ - New indexing schemes, New deltastreamer sources, New platform services.
+ - Any change that impacts the public interfaces of the project.
+
+It can also be used to describe large direction shifts to the project (e.g: Flink support) or new green field ideas (e.g Hudi for ML flows)
+Not all RFCs require the same effort and detail. For critical changes such as the File Format and Index, we need to deeply discuss the trade-offs of 
+making such a change and how it impacts current and new users. Any changes to such components affect the correctness of a dataset (backwards and forward versions supported). 
+Other changes such as code refactor might require more details around abstractions but as long as there is good test coverage, a migration plan section can be avoided. 
+It may happen that you are making a bunch of changes across many components to enable an already existing feature. For example, introducing a new config along with reporting metrics, 
+enhancing a tool and also improving documentation and the on-boarding experience. If all of these changes are linked to a general feature/idea, these can be grouped together under a single RFC.
+
+## Who should initiate the RFC ?
+Anyone can initiate an RFC. Please note that if you are unsure of whether a feature already exists or if there is a plan already to implement a similar one, always start a discussion thread on the dev mailing list before initiating a RFC. This will help everyone get the right context and optimize everyone’s usage of time.
+
+## How do you initiate the RFC ?
+
+### Proposing the RFC
+1. First, start a discussion thread on the Apache Hudi dev mailing list. The subject of the discussion thread can be `DISCUSS <proposed idea>` to create an RFC. 
+Use this discussion thread to get an agreement from people on the mailing list that your proposed idea necessitates an RFC.
+2. File an umbrella JIRA for the RFC with the label `hudi-umbrellas` added to it.
+3. Raise a PR, adding an entry to the table at `rfc/README.md`, picking the next available RFC number. Hudi committers will help land that.
+
+### Writing the RFC
+1. Create a folder `rfc-<number>` under `rfc` folder
+2. Copy the rfc template file `rfc/template.md` to `rfc/rfc-<number>.md` and proceed to draft your design document.
+3. [Optional] Place any images used by the same directory using the `![alt text](./image.png)` markdown syntax.
```

Comment:
better to keep each RFC in its own folder? `rfc/rfc-99/README.md` so it'll be browsable on github. also good to keep images in its own RFC folder.

Reply:
Oh, my bad. That's what I intended. Will make instructions clearer.

Reply:
ok sounds good.